tests/e2e/specs/features/04_tags.cy.js

```diff
@@ -2,9 +2,13 @@ describe("Tags CRUD", () => {
   const tagList = ["outdated", "priority", "lyon"];
 
   beforeEach(() => {
```

Comment:
Can we use an async function in beforeEach to use `await` instead of `.then()`?

Reply:
No, because here `then()` refer to a [Cypress command](https://docs.cypress.io/api/commands/then) not a promise.